Dockerfile

```diff
@@ -1,41 +1,59 @@
-FROM keboola/db-component-ssh-proxy:latest AS sshproxy
-FROM php:7.1-fpm
+FROM php:7.4-cli
 
-ENV DEBIAN_FRONTEND noninteractive
+ARG COMPOSER_FLAGS="--prefer-dist --no-interaction"
+ARG DEBIAN_FRONTEND=noninteractive
 ENV COMPOSER_ALLOW_SUPERUSER 1
 ENV COMPOSER_PROCESS_TIMEOUT 3600
-ARG COMPOSER_FLAGS="--prefer-dist --no-interaction"
-
-# Install dependencies
-RUN apt-get update -q \
-  && apt-get install default-mysql-client ssh git zip wget curl make patch unzip bzip2 time libzip-dev -y --no-install-recommends
 
-RUN docker-php-ext-install pdo_mysql
+WORKDIR /code/
 
-# use recommended php settings
 COPY docker/php-prod.ini /usr/local/etc/php/php.ini
+COPY docker/composer-install.sh /tmp/composer-install.sh
 
-# add debugger
+# Install dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
tady bych řekl že nebude potřeba instalovat všechny balíčky např.:
git
make
patch
bzip2
time
default-mysql-client

Reply:
myslim si, ze toto je trocha mimo ciel tohto PR, ale ok, checknem to. `git` pouziva composer.